Controller/Adminhtml/System/ProcessIntegrationToken.php

```diff
@@ -112,6 +117,9 @@ public function execute()
                     $boltMode = $this->boltConfigHelper->isSandboxModeSet($storeId) ? IntegrationManagement::BOLT_INTEGRATION_MODE_SANDBOX : IntegrationManagement::BOLT_INTEGRATION_MODE_PRODUCTION;
                     // Update related config.
                     $this->resourceConfig->saveConfig(BoltConfigHelper::XML_PATH_CONNECT_INTEGRATION_MODE, $boltMode);
+                    if (version_compare($this->boltConfigHelper->getStoreVersion(), '2.4.4', '>=')) {
+                        $this->resourceConfig->saveConfig(self::CONFIG_PATH_INTEGRATION_BEARER, true);
```

Comment:
This is quite convenient for us, but we silently change Magento setting which related to safety, so there is a lot of props and cons.
Do you mind to create a specific asana for this PR, explain what we want to do and ask Prathika opinion/approval here?